apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -21,23 +21,27 @@ export default function CurriculumQuickAssign({
   isNewSection,
   updateSection,
   sectionCourse,
+  initialParticipantType,
 }) {
   const [courseOfferings, setCourseOfferings] = useState(null);
   const [decideLater, setDecideLater] = useState(false);
   const [marketingAudience, setMarketingAudience] = useState(null);
   const [selectedCourseOffering, setSelectedCourseOffering] = useState();
 
-  const showPlOfferings = queryParams('participantType') !== 'student';
+  const participantType = isNewSection
+    ? queryParams('participantType')
+    : initialParticipantType;
+
+  const showPlOfferings = participantType !== 'student';
```

Comment:
can we use the `ParticipantAudience` constants from `@cdo/apps/generated/curriculum/sharedCourseConstants` for this?

Reply:
Updated! Thanks!